wrc/codegen/cg.py

```diff
@@ -21,3 +21,6 @@ def emit(self, ast_reg, ast_guide):
         if (ast_guide):
             self.visit(ast_guide)
         return (codegen_reg, self.codegen)
+
+    def shouldEmitSubsection(self, language, title):
+        return not (language == "english" and title == "Translation")
```

Comment:
Is it also easy to check that this section appears (exactly) once?

Reply:
hum it's not straightforward as the check passes in `sema/check.py` don't have knowledge about the language being parsed.

I could come up with something easy, and dirty to check this, but since this specific part (the "Translation" subsection in English) will be handled by the WRC I'm not sure it's worth it.

Reply:
Okay, that sounds fine. I've filed https://github.com/thewca/wca-regulations-compiler/issues/14 to keep track.